wikimedia/wikipedia-ios
# Talk pages - RTL fixes

**Phabricator:** 
https://phabricator.wikimedia.org/T301824 (feature-wide, so this is the epic)

### Notes
This PR does a few different things for RTL support:

1. A couple of slight tweaks to UI to have it display better in RTL. These are in the last two commits, 4a7f91e6 &  d562d6d7. The bugs I came across were in the subscribe / reply buttons (due to UIEdgeInsets) and a missing constraint in the header view.
2. A larger change (most of the commits) to match what our old talk pages were doing a little more closely. The goal here is to get the orientation of the _content_ to match what the orientation of the Wiki language is. I generally tried to leave any navigation (navigation buttons, toolbar buttons) alone. I did this by saving a preferred `semanticContentAttribute` in the `TalkPageViewModel` and adjusting various content views and `textAlignment` values to it.
3. Along these lines, design also requested that we pull localizations for the Wiki language for some native elements, again to try to have all of the content be consistently in the same language. This is in the commits 1b4a134b & f5d574a.

This PR is based on #4368, please merge after that is in.

### Test Steps
1. Run the app in a device LTR language in iOS Settings. Confirm everything looks aligned correctly for a LTR Wiki article and user talk page.
2. Visit an RTL article and user talk page. Confirm the content of the talk page switches to RTL. Confirm translations of some native elements (like the header view title) are in the RTL language (this test is a little tricky since some translations may not be in yet. We can lean on QA more for some of this.)
3. Now change device to RTL language in iOS Settings.  Confirm everything looks aligned correctly for a RTL article and user talk pages.
5. Visit an LTR article and user talk page. Confirm the content of the talk page switches to LTR.

### Screenshots/Videos

LTR device, LTR Wiki
![IMG_0841](https://user-images.githubusercontent.com/3620196/197297677-3c362770-95bc-4a76-9d52-37151af0d18a.PNG)

LTR device, RTL Wiki
![IMG_0842](https://user-images.githubusercontent.com/3620196/197297671-fd1029b1-fd51-4858-ac64-311299e96b75.PNG)

RTL device, RTL Wiki
![IMG_0843](https://user-images.githubusercontent.com/3620196/197297666-8f3cf1b2-10d5-4b5b-9402-9b1a5fc79955.PNG)

RTL device, LTR Wiki
![IMG_0844](https://user-images.githubusercontent.com/3620196/197297663-9e8c4665-a9e0-4da4-aff3-5f1e599d9d91.PNG)
